12/umbraco-cms/content-delivery-api.md

```diff
@@ -0,0 +1,413 @@
+  "properties": {
+    "property1Alias": "string",
+    "property2Alias": 0,
+    "property3Alias": true,
+    "property4Alias": [],
+    "property5Alias": {},
+    "property6Alias": null
+  },
+  "cultures": {
+    "cultureIsoCode1": {
+      "path": "string",
+      "startItem": {
+        "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+        "path": "string"
+      }
+    },
+    "cultureIsoCode2": {
+      "path": "string",
+      "startItem": {
+        "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+        "path": "string"
+      }
+    }
+  }
+}
+```
+
+</details>
+
+<details>
+
+<summary>Start item</summary>
+
+The start item indicates the root node to which a content item belongs. This concept is particularly useful for omitting the root node URL segment in the output, especially in cases where multiple Umbraco sites are configured under a single domain. The start item is represented by a header and is included in the JSON response of the Delivery API as the `startItem` property. This property contains both the item’s `id` and `path`. Note that the start item path segment will not be included in the value of the `path` JSON property.
+
+In Umbraco, `Umbraco.CMS.Global.HideTopLevelNodeFromPath` setting can be used to enable or disable including the top-level node in the URL of your content items. If the setting is set to `false`, you can still use the root node URL segment in the `path` parameter of the "_Get by route_" endpoint. However, the preferred approach is to make use of the `Start-Item` header to specify the URL segment or `id` of the root content node. This way, you can omit the root node URL segment from the content item path when querying for a content item. The `HideTopLevelNodeFromPath` setting does not affect the output URLs when rendering the content output.
+
+</details>
+
+<details>
+
+<summary>Output expansion</summary>
+
+**Output expansion** is a concept that refers to the ability to retrieve additional data about related content in the JSON output. By default, when a property that allows picking a different content item is used in a content node, for example, through a content picker property editor, the JSON representation of that content node will only contain a shallow representation of the referenced item. That means, only the basic information about the linked item, without the nested properties. However, with output expansion, it is possible to expand all or specific properties of the content, provided they are expandable and referenced at the first level.
+
+This feature can be used when querying for a content item or items through the Delivery API, by adding an `expand` parameter to the query. The value of this parameter can be either `"all"` to expand all properties on the content item, or `"property:alias, alias, alias"` to expand specific ones.
+
+
+
+The following snippet demonstrates the output of a "My post" content node without expanding the `linkedItem` content picker property.
+
```

Comment:
_The "suggest" feature does not like embedding code... but I'd suggest the same structure as below, without the expansion - just to show the request._

#### Request

```
GET
/umbraco/delivery/api/v1/content/item/9bdac0e9-66d8-4bfd-bba1-e954ed9c780d
```

#### Response

Reply:
So to have that structure as well before the default output?

Reply:
Yep 😄